db/migrate/20210518134556_create_prisons.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePrisons < ActiveRecord::Migration[6.0]
+  def change
+    create_table :prisons, id: false do |t|
+      t.string :prison_type, null: false
+      t.string :code, primary_key: true, null: false, index: {unique: true}
```

Comment:
You don't need the index or the uniqueness check now that code is the primary key.